application/Espo/Core/Utils/Database/Schema/Converter.php

```diff
@@ -102,13 +102,14 @@ class Converter
     protected $maxIndexLength = null;
 
     public function __construct(
-        Metadata $metadata,
-        FileManager $fileManager,
-        Schema $databaseSchema,
-        Config $config,
-        Log $log,
+        Metadata     $metadata,
+        FileManager  $fileManager,
+        Schema       $databaseSchema,
+        Config       $config,
+        Log          $log,
```

Comment:
Would be good if automatic argument aligning and other code style fixes wouldn't have been enabled in IDE. So only actual changes were in PRs.

Reply:
Sorry about that.